frontend/fingertips-frontend/mock/server/handlers.ts

```diff
@@ -32,9 +32,10 @@ export const handlers = [
 
     return HttpResponse.json(...resultArray[next() % resultArray.length]);
   }),
-  http.get(`${baseURL}/indicators/:indicatorId/data`, async () => {
+  http.get(`${baseURL}/indicators/:indicatorId/data`, async ({ params }) => {
+    const indicatorId = params.indicatorId?.toString() ?? '1';
```

Comment:
It might be slightly cleaner to make this:
```suggestion
    const indicatorId = params.indicatorId;
    if (typeof indicatorId !== 'string') {
      return HttpResponse.json(getGetHealthDataForAnIndicator400Response(), {
        status: 400,
      });
    }
```

This way it returns an HTTP 400 error when no indicator ID is provided (which is an invalid API request) and we don't have to assume a default value of indicator ID just to satisfy the type system.

Reply:
Good suggestion - thanks